src/client/components/LastMessageList/LastMessagesList.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { object } from '@storybook/addon-knobs';
+import './LastMessagesList.css';
+
+export default function LastMessageList({ lastChannels }) {
+  if (!lastChannels) {
+    return <p>No channels, start a conversation to see it here</p>;
+  }
+  return (
+    <ul>
+      {lastChannels.map(({ message, updated_at, fk_channel_id, title }) => (
```

Comment:
I would prefer if you renamed these in the controller to a nice camelCase alternative and then use those in frontend.

Reply:
it is done.